Makefile

```
@@ -490,6 +493,10 @@ fixtures/rpm-signed: fixtures gnupghome
 fixtures/rpm-unsigned: fixtures
 	rpm/gen-fixtures.sh $@ rpm/assets
 
+fixtures/rpm-unsigned-meta-only: fixtures fixtures/rpm-unsigned
+	mkdir -p $@
+	cp -r fixtures/rpm-unsigned/repodata $@/
```

I believe the `-r` option for cp is discouraged.

```suggestion
	cp -R fixtures/rpm-unsigned/repodata $@/
```

---

any mentioned in ```man``` but for consistency of ```Makefile``` looks much better.

---

The behavior for `-r` is not defined by POSIX and can vary from system to system. On some systems, it's equivalent to `-R` but on other systems (e.g. FreeBSD) it's equivalent to `-RL`.

---

thank you